modules/upgrade-cli.adoc

```diff
@@ -8,39 +8,42 @@
 = Upgrading your cluster through the CLI
 
 
-
 You can use the OpenShift Cluster Manager (OCM) CLI upgrade wizard to schedule automatic and manual upgrade policies, list the upgrade policy, and delete the upgrade policy for your {product-title} cluster.
 
+// TODO: Should there be a prereq here to have the ocm CLI installed? This is the first time I've come across it
+
+// TODO: Suggest another look at this procedure. Some of it looks like it might be sequential (like kicking off with the "ocm" command, but then next it'd be  choice of manual or automatic. And then it just gives some details about listing and deleting, which aren't necessarily a required part of the procedure. Also a question on the bold - does the command actually pop up a GUI wizard, or just walk you through the steps in the terminal?
```

Comment:
Just in the terminal.

Reply:
Okay then yeah I would probably avoid the use of bold in the procedure then.